numpy/core/src/common/simd/neon/arithmetic.h

```diff
@@ -133,10 +133,33 @@
 
 // Horizontal add: Calculates the sum of all vector elements.
 #if NPY_SIMD_F64
+    #define npyv_sum_u8 vaddvq_u8
+    #define npyv_sum_u16 vaddvq_u16
```

Comment:
```suggestion
    #define npyv_sumup_u8  vaddlvq_u8
    #define npyv_sumup_u16 vaddlvq_u16
```
`vaddvq_*` intrinsics wrap around the overflow, we need `vaddlvq_*` instead.